module/Company/view/company/admin/edit-job.phtml

```diff
@@ -174,6 +174,19 @@ $element->setAttribute('class', 'form-control');
 <?php endforeach;?>
 </div>
 
+<?php
+$element = $form->get('labels');
+?>
+<div class="form-group<?= count($element->getMessages()) > 0 ? ' has-error' : '' ?>">
+    <label for="<?= $element->getName() ?>" class="control-label col-sm-2">
+        <?= $element->getOption('label') ?>
+    </label>
+    <div class="col-sm-10">
+        <?= $this->formMultiCheckbox($element) ?>
```

Comment:
I agree that more user-friendly options exist. Yet Zend does not allow me to easily modify the way this element is rendered. I also do not see how to use a decorator with this specific element.

Reply:
@wsinnema, you've worked with Zend Forms in the past for the company module. Do you know if it's possible to add breaks in a `MultiCheckbox` element? See code [here](https://github.com/GEWIS/gewisweb/pull/964/files/86ca2566a94d376937e245ea4b3736837fe1d6ac#diff-037cb76f3f5ebdbff79ef16fef76a3d2R42-R49).

Reply:
I think this can be done by using `setLabelAttributes()`. Just add a class with `margin-right` (and `bottom`?) to give the elements a bit more space. I'll try to see if this works.